hackclub.com.yaml

```
@@ -1642,6 +1642,12 @@ nust:
   - ttl: 1
     type: CNAME
     value: nusthackclub.netlify.app.
+ocsa:
+  - ttl: 1
+    type: TXT
+    values:
+      - google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
+    
```

Looks like you're missing a CNAME record -- this won't direct anyone to your site, just verify that you have the subdomain.

---

Yup, you can still use TXT records, but this looks incomplete. A TXT record won't actually connect `ocsa.hackclub.com` to Google Sites.

```suggestion
ocsa:
  - ttl: 1
    type: TXT
    values:
      - google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
  - ttl: 300
    type: CNAME
    values:
      - ghs.googlehosted.com.
```

This suggestion should work -- see https://www.ionos.ca/digitalguide/domains/domain-tips/google-sites-with-custom-domain/ for a source

---

sure we can try that

---

Seems like it didn't pass the test

---

I think we should try with just the txt record, and if it doesn't work, I can make further pull requests. I think how it works is google sites first verifies the Domain with the txt record, them provides instructions for a cname or something else